java/chainguard-25/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM europe-north1-docker.pkg.dev/cgr-nav/pull-through/nav.no/jre:openjdk-25
+
+ENV LANG='nb_NO.UTF-8' LANGUAGE='nb_NO:nb' LC_ALL='nb_NO.UTF-8' TZ="Europe/Oslo"
+
+ENV JDK_JAVA_OPTIONS="-XX:+PrintCommandLineFlags \
+                      -XX:ActiveProcessorCount=2 \
```

Comment:
Det er kanskje lurt å øke ActiveProcessorCount på java25-byggene for å få flere carrier threads for virtuelle tråder

Reply:
Ja, det er jo et fint rundt tall :-D